auth0/v3/authentication/base.py

```diff
@@ -19,8 +19,8 @@ def _process_response(self, response):
         except ValueError:
             return response.text
         else:
-            if 'error' in text:
-                raise Auth0Error(status_code=text['error'],
-                                 error_code=text['error'],
-                                 message=text['error_description'])
+            if response.status_code >= 400:
```

Comment:
Could you handle the case where [for some reason](https://github.com/requests/requests/blob/master/requests/adapters.py#L265) `status_code` is Null. Also If you can add the test for this case, it will be great.

Reply:
@saltukalakus Ok no problem, I've added the code change. Let me know if all is ok.